src/apps/dashboard/routes/index.tsx

```diff
@@ -0,0 +1,178 @@
+import ItemCountsWidget from '../components/widgets/ItemCountsWidget';
+import { useItemCounts } from '../features/metrics/api/useItemCounts';
+
+export const Component = () => {
+    const theme = useTheme();
+    const isMedium = useMediaQuery(theme.breakpoints.only('md'));
+    const isExtraLarge = useMediaQuery(theme.breakpoints.only('xl'));
+    const [ isRestartConfirmDialogOpen, setIsRestartConfirmDialogOpen ] = useState(false);
+    const [ isShutdownConfirmDialogOpen, setIsShutdownConfirmDialogOpen ] = useState(false);
+    const startTask = useStartTask();
+    const restartServer = useRestartServer();
+    const shutdownServer = useShutdownServer();
+
+    const { data: tasks, isPending: isTasksPending } = useLiveTasks({ isHidden: false });
+    const { data: devices } = useLiveSessions();
+
+    const dayBefore = useMemo(() => (
+        subSeconds(new Date(), 24 * 60 * 60).toISOString()
+    ), []);
+
+    const weekBefore = useMemo(() => (
+        subSeconds(new Date(), 7 * 24 * 60 * 60).toISOString()
+    ), []);
+
+    const { data: logs, isPending: isLogsPending } = useLogEntries({
+        startIndex: 0,
+        limit: 7,
+        minDate: dayBefore,
+        hasUserId: true
+    });
+
+    const { data: alerts, isPending: isAlertsPending } = useLogEntries({
+        startIndex: 0,
+        limit: 4,
+        minDate: weekBefore,
+        hasUserId: false
+    });
+
+    const { data: systemStorage, isPending: isSystemStoragePending } = useSystemStorage();
+    const { data: systemInfo, isPending: isSystemInfoPending } = useSystemInfo();
+    const { data: itemCounts, isPending: isItemCountsPending } = useItemCounts();
+
+    const promptRestart = useCallback(() => {
+        setIsRestartConfirmDialogOpen(true);
+    }, []);
+
+    const closeRestartDialog = useCallback(() => {
+        setIsRestartConfirmDialogOpen(false);
+    }, []);
+
+    const promptShutdown = useCallback(() => {
+        setIsShutdownConfirmDialogOpen(true);
+    }, []);
+
+    const closeShutdownDialog = useCallback(() => {
+        setIsShutdownConfirmDialogOpen(false);
+    }, []);
+
+    const onScanLibraries = useCallback(() => {
+        const scanLibrariesTask = tasks?.find((value) => value.Key === 'RefreshLibrary');
+
+        if (scanLibrariesTask && scanLibrariesTask.Id) {
+            startTask.mutate({
+                taskId: scanLibrariesTask.Id
+            });
+        }
+    }, [ startTask, tasks ]);
+
+    const onRestartConfirm = useCallback(() => {
+        restartServer.mutate();
+        setIsRestartConfirmDialogOpen(false);
+    }, [ restartServer ]);
+
+    const onShutdownConfirm = useCallback(() => {
+        shutdownServer.mutate();
+        setIsShutdownConfirmDialogOpen(false);
+    }, [ shutdownServer ]);
+
+    const isPending = isLogsPending || isAlertsPending || isSystemStoragePending
+        || isSystemInfoPending || isTasksPending || isItemCountsPending;
+
+    if (isPending) {
```

Comment:
We shouldn't block the page from loading until all the data is available. Components should render placeholders/skeletons until the data is available.